tests/locators_people_page.py

```diff
@@ -30,11 +28,12 @@ class PeoplePageLocator:
                             TABLE_USER_ID_USERS, TABLE_NAME_USERS, TABLE_ICON_LINK, TABLE_ICON_LINK_2]
 
 
-class LoginLocator:
-    USER = 'j_username'
-    PASSWD = 'j_password'
-    SUBMIT = 'Submit'
-
-
 class URLLocators:
     URL_PEOPLE = TestData.BASE_URL + 'asynchPeople/'
+
+
+class IdUsersLocator:
+    TABLE_USER_ID_ARROW = (By.XPATH, '//*[@id="people"]/tbody/tr[1]/th[2]/a/span')
+    locator_id_user_arrow = [TABLE_USER_ID_ARROW]
+    TABLE_USER_ID = (By.XPATH, '//*[@id="people"]/tbody/tr[1]/th[2]/a')
+    locator_id_user = [TABLE_USER_ID]
```

Comment:
what was the reason to create these variables "locator_id_user_arrow" and "locator_id_user"? 
i don't see where you using them.

Reply:
This is for future tests